WebViewControl/WebView.cs

```diff
@@ -372,7 +366,7 @@ public partial class WebView : IDisposable {
             }
 
             if (interceptCall == null) {
-                interceptCall = target => target();
+                interceptCall = target => target.Invoke();
```

Comment:
why?

Reply:
Probably due to changes to fix the listeners test (problem was in CefGlue's side)